index.html

```diff
@@ -57,17 +65,17 @@ <h2 class="header-subtitle">I’m a Software Developer</h2>
             </a>
           </li>
         </ul>
-      </section>
+      </div>
     </section>
 
     <section class="recent-container" id="porto">
-      <div class="work-container">
-        <h2 class="recent-work">My Recent Works</h2>
-        <hr>
-      </div>
-      <div class="image-p">
-        <img class="image-placeholder" src="Images/skeloton/placeholder.png" alt="gymfit picture with beutifull girl">
-        <div class="multi">
+       <div class="work-container">
+         <h2 class="recent-work">My Recent Works</h2>
+         <hr class="re-line">
+       </div>
+       <div class="image-p">
+         <img class="image-placeholder" src="Images/skeloton/placeholder.png" alt="gymfit picture with beutifull girl">
+         <div class="multi">
```

Comment:
- [ ] Kindly style the contents of this container to match the design template. You are required to stick to the design as much as possible, kindly fix the spacings to match the design
## Your design
![r1](https://user-images.githubusercontent.com/94551484/186988868-2d1eb78f-644f-4bfb-8a31-0afd4b516275.png)
## Figma template
![s1](https://user-images.githubusercontent.com/94551484/186988874-4bf4bf52-aa95-43c1-9bce-77d6952fc916.png)
.

Reply:
I kindly advise you to follow the previous reviewer's advice.

The title of the section is not the same font, and the language tags do not adapt to the size of the text or are not centered, (I recommend you not to use fixed widths, instead just use padding)